### PR TITLE
make hue-light compliant with JCasC

### PR DESCRIPTION
**Pull request by ewelinawilkosz**
Changes introduces in this PR makes hue-light-plugin compliant with https://github.com/jenkinsci/configuration-as-code-plugin

had to bump jenkins version to make it possible to use DataBoundSetter

cc @ndeloof

### CODE REVIEW

File: src/main/java/org/jenkinsci/plugins/hue_light/LightNotifier.java [hunk 1]
```diff
@@ -23,15 +23,16 @@
...
-    private static final String FORM_KEY_BLUE = "colorBlue";
-    private static final String FORM_KEY_GREEN = "colorGreen";
-    private static final String FORM_KEY_YELLOW = "colorYellow";
-    private static final String FORM_KEY_RED = "colorRed";
+    private static final String FORM_KEY_BLUE = "blue";
+    private static final String FORM_KEY_GREEN = "green";
+    private static final String FORM_KEY_YELLOW = "yellow";
+    private static final String FORM_KEY_RED = "red";
     private static final String FORM_KEY_SATURATION = "saturation";
     private static final String FORM_KEY_BRIGHTNESS = "brightness";
```
Comment (ndeloof) [inline src/main/java/org/jenkinsci/plugins/hue_light/LightNotifier.java]: AFAICT you don't need those constants anymore

**Comment by ewelinawilkosz** [comment]
@MMore this is not an urgent issue, but can you please let me know if you're still maintaining the plugin and if we can expect this PR to be a part of next release in the foreseeable future? 
thanks in advance!

**Comment by oleg-nenashev** [comment]
It also makes sense to add Jenkinsfile:

```
// Build the plugin using https://github.com/jenkins-infra/pipeline-library
buildPlugin(jenkinsVersions: [null, "2.107.1"])
```

**Comment by MMore** [comment]
@ewelinawilkosz Sorry for the delay and thanks for your contribution. Right now I don't have the possibility to test your change. Would you like to integrate @oleg-nenashev suggestions?

**Comment by oleg-nenashev** [comment]
@ewelinawilkosz gentle ping ':)

**Comment by ewelinawilkosz** [comment]
thanks for the ping @oleg-nenashev - I applied your suggestions

**Pull request merged**